packages/turf-kinks/index.ts

```diff
@@ -40,114 +41,16 @@ export default function kinks<T extends LineString | MultiLineString | Polygon |
         type: "FeatureCollection",
         features: [],
     };
-    if (featureIn.type === "Feature") {
-        feature = featureIn.geometry;
-    } else {
-        feature = featureIn;
-    }
-    if (feature.type === "LineString") {
-        coordinates = [feature.coordinates];
-    } else if (feature.type === "MultiLineString") {
-        coordinates = feature.coordinates;
-    } else if (feature.type === "MultiPolygon") {
-        coordinates = [].concat.apply([], feature.coordinates);
-    } else if (feature.type === "Polygon") {
-        coordinates = feature.coordinates;
-    } else {
+
+    // @ts-ignore
```

Comment:
Why is there a @ts-ignore here?

Reply:
Probably because I don't understand ts :)

Reply:
`This condition will always return 'false' since the types '"LineString" | "MultiLineString" | "Polygon" | "MultiPolygon" | "Feature"' and '"Point"' have no overlap.` This is saying that because of the typings for `featureIn` it wouldn't actually expect `Point` to ever come in. This makes sense if you're a typescript consumer but isn't necessarily safe if you're a javascript consumer. You can cast featureIn to allow the checks like so:

```
// at the top
import {GeometryObject} from "@turf/helpers";
// down below
(featureIn as GeometryObject | Feature<GeometryObject>).type === "Point"
```